core/include/mmcore/view/AbstractView.h

```diff
@@ -115,56 +116,43 @@ class MEGAMOLCORE_API AbstractView : public Module, public megamol::frontend_res
     virtual bool IsParamRelevant(const vislib::SmartPtr<param::AbstractParam>& param) const;
 
     /**
-     * Answer the camera synchronization number.
+     * Set the camera for this view externally
      *
-     * @return The camera synchronization number
+     * @param camera A fully intialized camera to use for rendering the view
+     * @param isMutable Tell the view whether it can modify, i.e. control, the camera or not
      */
-    virtual unsigned int GetCameraSyncNumber(void) const = 0;
+    virtual void SetCamera(Camera camera, bool isMutable = true);
 
     /**
-     * Renders this AbstractView.
-     *
-     * @param context The context information like time or GPU affinity.
-     */
-    void Render(const mmcRenderViewContext& context) {
-        this->Render(context, nullptr);
-    };
+    * ...
+    */
+    virtual void CalcCameraClippingPlanes(float border);
 
     /**
-     * Renders this AbstractView. Implemented by child classes.
+     * Renders this AbstractView.
+     * The View will use its own camera and framebuffer for the rendering exectuion
      *
-     * @param context The context information like time or GPU affinity.
+     * @param time ...
+     * @param instanceTime ...
      */
-    virtual void Render(const mmcRenderViewContext& context, Call* call) = 0;
+    using ImageWrapper = megamol::frontend_resources::ImageWrapper;
+    virtual ImageWrapper Render(double time, double instanceTime) = 0;
+
+    virtual ImageWrapper GetRenderingResult() const = 0;
```

Comment:
If `Render()` already returns the Image we dont need `GetRenderingResult()` anymore?

Reply:
I think I kept this around in case someone needs to access the rendering result after the call to `Render()` was already made elsewhere in the code, e.g. when a view is called by another view via Call callback. But at the moment I don't thinks it's possible for one view to directly call a function with return value of another view anyway...